Discard example broken

### Expected behavior

Run the discard example in two terminals:
./run-example discard-server  /  ./run-example discard-client

should work

### Actual behavior
Crash with:

Error: Could not find or load main class 
Caused by: java.lang.ClassNotFoundException: 

Tracked this down to example/pom.xml where the last part where the example is run has a argument ${argLine.alpnAgent}.
This argument is empty and java do not like an empty arg. It works if it is removed.

Then the ssl part is broken.
Running the commands (with the alpnAgent removed)

./run-example -Dssl discard-server  /  ./run-example -Dssl discard-client
The example just hangs until the handshake times out.

Tracked this down to the DiscardClient.java. It uses ServerUtil.buildSslContext(), but this is a server setup.

The client can be changed to:

    private static final boolean SSL = System.getProperty("ssl") != null;

    public static void main(String[] args) throws Exception {
        // Configure SSL.
        final SslContext sslCtx = SSL ? SslContextBuilder
            .forClient()
            .trustManager(InsecureTrustManagerFactory.INSTANCE)
            .build() : null;

### Steps to reproduce
Just build and run the example.

### Minimal yet complete reproducer code (or URL to code)
N/A

### Netty version
4.1.115

### JVM version (e.g. `java -version`)
N/A

### OS version (e.g. `uname -a`)
Linux fedora 6.9.7-200.fc40.x86_64 #1 SMP PREEMPT_DYNAMIC Thu Jun 27 18:11:45 UTC 2024 x86_64 GNU/Linux

--- a/example/src/main/java/io/netty/example/discard/DiscardClient.java
+++ b/example/src/main/java/io/netty/example/discard/DiscardClient.java
@@ -23,21 +23,29 @@ import io.netty.channel.EventLoopGroup;
 import io.netty.channel.nio.NioEventLoopGroup;
 import io.netty.channel.socket.SocketChannel;
 import io.netty.channel.socket.nio.NioSocketChannel;
-import io.netty.example.util.ServerUtil;
 import io.netty.handler.ssl.SslContext;
+import io.netty.handler.ssl.SslContextBuilder;
+import io.netty.handler.ssl.util.InsecureTrustManagerFactory;
 
 /**
  * Keeps sending random data to the specified address.
  */
 public final class DiscardClient {
 
+    static final boolean SSL = System.getProperty("ssl") != null;
     static final String HOST = System.getProperty("host", "127.0.0.1");
     static final int PORT = Integer.parseInt(System.getProperty("port", "8009"));
     static final int SIZE = Integer.parseInt(System.getProperty("size", "256"));
 
     public static void main(String[] args) throws Exception {
         // Configure SSL.
-        final SslContext sslCtx = ServerUtil.buildSslContext();
+        final SslContext sslCtx;
+        if (SSL) {
+            sslCtx = SslContextBuilder.forClient()
+                .trustManager(InsecureTrustManagerFactory.INSTANCE).build();
+        } else {
+            sslCtx = null;
+        }
 
         EventLoopGroup group = new NioEventLoopGroup();
         try {

